timeseries/src/autogluon/timeseries/models/chronos/chronos2.py

```diff
@@ -0,0 +1,329 @@
+    .. [Ansari2025] Ansari, Abdul Fatir, Shchur, Oleksandr, Kuken, Jaris et al.
+        "Chronos-2: From Univariate to Universal Forecasting." (2025).
+        https://arxiv.org/abs/2510.15821
+
+    Other Parameters
+    ----------------
+    model_path : str, default = "autogluon/chronos-2"
+        Model path used for the model, i.e., a HuggingFace transformers ``name_or_path``. Can be a
+        compatible model name on HuggingFace Hub or a local path to a model directory.
+    batch_size : int, default = 256
+        Size of batches used during inference.
+    device : str, default = None
+        Device to use for inference (and fine-tuning, if enabled). If None, model will use the GPU if
+        available.
+    context_length : int or None, default = None
+        The context length to use in the model. If None, the model will use its default context length
+        of 8192. Shorter context lengths will decrease model accuracy, but result in faster inference.
+    fine_tune : bool, default = False
+        If True, the pretrained model will be fine-tuned.
+    fine_tune_mode : str, default = "lora"
+        Fine-tuning mode, either "full" for full fine-tuning or "lora" for Low Rank Adaptation (LoRA).
+        LoRA is faster and uses less memory.
+    fine_tune_lr : float, default = 1e-5
+        The learning rate used for fine-tuning. When using full fine-tuning, a lower learning rate such as 1e-6
+        is recommended.
+    fine_tune_steps : int, default = 1000
+        The number of gradient update steps to fine-tune for.
+    fine_tune_batch_size : int, default = 256
+        The batch size to use for fine-tuning.
+    eval_during_fine_tune : bool, default = False
+        If True, validation will be performed during fine-tuning to select the best checkpoint.
+        Setting this argument to True may result in slower fine-tuning.
+    fine_tune_eval_max_items : int, default = 256
+        The maximum number of randomly-sampled time series to use from the validation set for evaluation
+        during fine-tuning. If None, the entire validation dataset will be used.
+    fine_tune_lora_config : dict, optional
+        Configuration for LoRA fine-tuning when ``fine_tune_mode="lora"``. If None and LoRA is enabled,
+        a default configuration will be used. Example: ``{"r": 8, "lora_alpha": 16}``.
+    fine_tune_trainer_kwargs : dict, optional
+        Extra keyword arguments passed to ``transformers.TrainingArguments``
+    """
+
+    ag_model_aliases = ["Chronos-2"]
+
+    _supports_known_covariates = True
+    _supports_past_covariates = True
+
+    def __init__(
+        self,
+        freq: str | None = None,
+        prediction_length: int = 1,
+        path: str | None = None,
+        name: str | None = None,
+        eval_metric: str | None = None,
+        hyperparameters: dict[str, Any] | None = None,
+        **kwargs,
+    ):
+        hyperparameters = hyperparameters if hyperparameters is not None else {}
```

Comment:
This line can be removed since we are not using the `hyperparameters` during `__init__`